RELEASE.md

```diff
@@ -0,0 +1,15 @@
+# Instructions for making a release
+
+1. Change the version number in `package.json`. Run the following command in the package root directory, replacing <update_type> with one of the semantic versioning release types (prerelease, prepatch, preminor, premajor, patch, minor, major):
+
+```
+npm version <update_type> --preid pre --no-git-tag-version
+```
+
```

Comment:
let's add some of the details from your last comment to the docs, so that they don't get lost and we don't forget how to cut a release :smile: 
```suggestion

`--preid` specifies which suffix to use in the release such as `pre`, `next`, `beta`, `rc`, etc.

`prepatch`, `preminor`, and `premajor` start a new series of pre-releases while bumping the patch, minor, or major version. E.g. `premajor` with `--preid pre` would do a prerelease for a new major using the `-pre` suffix (i.e. it would be a new major with `-pre.0`)
```